packages/ember-data/lib/system/record-array-manager.js

```diff
@@ -153,7 +153,7 @@ export default Ember.Object.extend({
     for (var i = 0, l = records.length; i < l; i++) {
       record = records[i];
 
-      if (!get(record, 'isDeleted') && !get(record, 'isEmpty')) {
+      if (!record.isDeleted() && !record.isEmpty()) {
```

Comment:
This will be a breaking change for people currently using .get in their code. One we could probably make an Ember Watson transform for, but still very important to take into consideration and deprecate the getter form.